application.go

```diff
@@ -109,6 +129,28 @@ type DeleteAppOpts struct {
 	Force bool `url:"force,omitempty"`
 }
 
+// Diego ======
+
+// SetIPAddressPerTask defines that the application will have a IP address defines by a external agent. this
+// configuration is not allowed to be used with Port or PortDefinitions.
+func (r *Application) SetIPAddressPerTask(ipAddressPerTask *IPAddressPerTask) *Application {
+
+	r.Ports = make([]int, 0)
```

Comment:
The API describes that using ports/ports definition is not allowed if using IP-per-task. Actualy if you send a post with both options you'll get an error. I was asking myself to put or not this cleanup, it is like reproduce an API behavior in the client library.